jina/types/sets/chunk.py

```diff
@@ -48,17 +56,41 @@ def append(self, document: 'Document', **kwargs) -> 'Document':
         chunk.update_content_hash()
         return chunk
 
+    def extend(self, iterable: Iterable['Document']) -> None:
```

Comment:
I added the sibling update functionality to the extend method. It is a bit inconsistent because append does not update the siblings. But if we would update this attribute for append as well it could slow down the system when having many chunks. We could argue that the system is slowed down anyways if we have a lot of chunks and have the `doc.siblings = len(self)` also in the append. 
Anyways, I think there is the goal to remove siblings in the long term. So thinking about all the cases might not be necessary